src/coreclr/jit/morph.cpp

```diff
@@ -2787,6 +2787,14 @@ void CallArgs::AddFinalArgsAndDetermineABIInfo(Compiler* comp, GenTreeCall* call
                 {
                     if ((size > 1) && ((intArgRegNum + 1) == maxRegArgs) && (nextOtherRegNum == REG_STK))
                     {
+#if FEATURE_ARG_SPLIT
```

Comment:
Since LA requires `FEATURE_ARG_SPLIT`, this `#if` seems redundant.

Reply:
OK, Thanks